src/forms.py

```diff
@@ -0,0 +1,143 @@
+                        "questionItem": {
+                            "question": {
+                                "scaleQuestion": {
+                                    "low": low_val,
+                                    "high": high_val,
+                                    "lowLabel": low_label,
+                                    "highLabel": high_label,
+                                }
+                            }
+                        },
+                    },
+                    "location": {"index": idx},
+                }
+            }
+            for idx, title in enumerate(items)
+        ]
+        + [
+            {
+                "createItem": {
+                    "item": {
+                        "title": "Comments/Suggestions/Jokes",
+                        "questionItem": {
+                            "question": {
+                                "textQuestion": {
+                                    "paragraph": True,
+                                }
+                            }
+                        },
+                    },
+                    "location": {"index": len(items)},
+                }
+            }
+        ]
+    }
+
+    # get form service
+    form_service, drive_service = get_google_services()
+    if form_service is None or drive_service is None:
+        return None
+
+    # Creates the initial form
+    form_info = form_service.forms().create(body=form_info).execute()
+    form_id = form_info["formId"]
+    form_url = form_info["responderUri"]
+
+    # Add content to the form
+    form_service.forms().batchUpdate(formId=form_id, body=form_update).execute()
+
+    # Prepend image in separate update in case it fails
+    if image_url:
+        try:
+            image_update = {
+                "requests": [
+                    {
+                        "createItem": {
+                            "item": {
+                                "imageItem": {
+                                    "image": {
+                                        "altText": "It's bust time, baby.",
+                                        "sourceUri": image_url,
+                                    }
+                                }
+                            },
+                            "location": {"index": 0},
+                        }
+                    }
+                ]
+            }
+            form_service.forms().batchUpdate(
+                formId=form_id, body=image_update
+            ).execute()
+        except Exception as e:
+            print("Error adding image to form: ", e)
+
+    # Move form to correct folder + rename
+    file = (
+        drive_service.files()
+        .get(fileId=form_id, fields="capabilities/canMoveItemWithinDrive, parents")
+        .execute()
+    )
+    if file["capabilities"]["canMoveItemWithinDrive"]:
+        file_parent_id = file["parents"][0]
+        # TODO: Command fails silently if config.google_form_folder is invalid
```

Comment:
I probably can, my guess is that it would add another second of query time. I'm guessing actually the return value of this function will tell me if an error occurred, so I'll look into that

Reply:
It now prints an error by catching an exception. I'm not actually sure why the exception didn't just dump to screen when I tried this before.